### ISSUE TITLE
error generating code: call: module[] function[_start] failed: wasm error: unreachable

### ISSUE THREAD

**Issue by roccho-dev** [orisano/sqlc-gen-typescript-d1#5]
Hi.
I want to hear you when you check this error.

```sh
$ sqlc version
v1.26.0
```

```sh
$ sqlc generate
# package ts-d1
error generating code: call: module[] function[_start] failed: wasm error: unreachable
wasm stack trace:
        .runtime.runtimePanicAt(i32,i32)
        .runtime.slicePanic()
        .main.toUpperCamel(i32,i32,i32)
        .(main.Naming).toModelTypeName(i32,i32)
        ._start()
```

```json
{
	"version": "2",
	"plugins": [
		{
			"name": "ts-d1",
			"wasm": {
				"url": "https://github.com/orisano/sqlc-gen-ts-d1/releases/download/v0.0.0-a/sqlc-gen-ts-d1.wasm",
				"sha256": "a3d312f03e80d6530ea7be50c1b39cf2e34cfa846d91751973ed078cb2cddecc"
			}
		}
	],
	"sql": [
		{
			"schema": "./db/schema.sql",
			"queries": "./db/query.sql",
			"engine": "sqlite",
			"codegen": [
				{
					"out": "src/db",
					"plugin": "ts-d1",
					"options": {
						"workers-types": "experimental"
					}
				}
			]
		}
	]
}
```

'sha256' was provided your README 'curl' command.
Thanks.

**Comment by voluntas** [comment]
@PorcoRosso85 Please provide the minimal schema.sql and query.sql needed to reproduce the issue.

**Comment by roccho-dev** [comment]
Following occurs this error '--here'
After removing it, generation worked.

```sql
CREATE TABLE IF NOT EXISTS "schema_migrations" (version varchar(128) primary key);

-- here
CREATE TABLE _cf_KV (
      key TEXT PRIMARY KEY,
      value BLOB
    ) WITHOUT ROWID;


CREATE TABLE user (
    id INTEGER PRIMARY KEY AUTOINCREMENT,
    email VARCHAR(255) NOT NULL,
    user_name VARCHAR(255) NOT NULL,
    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
);
CREATE TABLE relation (
    id SERIAL PRIMARY KEY,
    ancester_user_id INT NOT NULL,
    descendant_user_id INT NOT NULL,
    path_depth INT NOT NULL,
    FOREIGN KEY (ancester_user_id) REFERENCES user (id),
    FOREIGN KEY (descendant_user_id) REFERENCES user (id)
);
CREATE TABLE contract (
    id SERIAL PRIMARY KEY,
    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
    contract_status VARCHAR(255) NOT NULL,
    contract_content JSON NOT NULL,
    relation_id INT NOT NULL,
    FOREIGN KEY (relation_id) REFERENCES relation (id)
);
-- Dbmate schema migrations
INSERT INTO "schema_migrations" (version) VALUES
  ('20240731053912');
```

**Comment by orisano** [comment]
@PorcoRosso85 Thanks for the report!
It appears that toUpperCamel panics when the table name begins with _.
What should be the type name of _cf_KV?

**Comment by roccho-dev** [comment]
@orisano cc: @voluntas 
Thanks for your checking my issue.

Sorry for unavailable to answer what type should be.
'_cf_KV' will generate automatically when we migrate 'cloudflare D1' with 'dbmate'.

https://github.com/amacneil/dbmate

**Comment by orisano** [comment]
I believe it has been corrected in 82f35cb7c0d7.
Please try it out as we are updating the release.

**Comment by roccho-dev** [comment]
@orisano 

Thanks, yours seems working well.
If anything will occur, I'll inform you again.
